src/Compilers/VisualBasic/Portable/BoundTree/BoundCall.vb

```diff
@@ -19,13 +39,31 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Optional hasErrors As Boolean = False
         )
             Me.New(syntax, method, methodGroupOpt, receiverOpt, arguments,
+                   defaultArguments:=BitVector.Null,
                    constantValueOpt,
                    isLValue:=method.ReturnsByRef,
                    suppressObjectClone:=suppressObjectClone,
                    type:=type,
                    hasErrors:=hasErrors)
         End Sub
 
+        Public Sub New(syntax As SyntaxNode,
+                       method As MethodSymbol,
+                       methodGroupOpt As BoundMethodGroup,
+                       receiverOpt As BoundExpression,
+                       arguments As ImmutableArray(Of BoundExpression),
+                       constantValueOpt As ConstantValue,
+                       isLValue As Boolean,
+                       suppressObjectClone As Boolean,
+                       type As TypeSymbol,
+                       Optional hasErrors As Boolean = False)
+            Me.New(syntax, method, methodGroupOpt, receiverOpt, arguments, defaultArguments:=BitVector.Null, constantValueOpt, isLValue, suppressObjectClone, type, hasErrors)
+        End Sub
+
+        Public Function Update(method As MethodSymbol, methodGroupOpt As BoundMethodGroup, receiverOpt As BoundExpression, arguments As ImmutableArray(Of BoundExpression), constantValueOpt As ConstantValue, isLValue As Boolean, suppressObjectClone As Boolean, type As TypeSymbol) As BoundCall
```

Comment:
>Public Function Update(meth [](start = 8, length = 27)

Please break this long line

Reply:
No longer relevant.

---
In reply to: [190018757](https://github.com/dotnet/roslyn/pull/26781#discussion_r190018757) [](ancestors = 190018757)